src/FsCodec.SystemTextJson/Options.fs

```diff
@@ -57,14 +57,19 @@ type Options private () =
             /// <summary>Apply <c>TypeSafeEnumConverter</c> if possible. Defaults to <c>false</c>.</summary>
             [<Optional; DefaultParameterValue(null)>] ?autoTypeSafeEnumToJsonString : bool,
             /// <summary>Apply <c>UnionConverter</c> for all Discriminated Unions, if <c>TypeSafeEnumConverter</c> not possible. Defaults to <c>false</c>.</summary>
-            [<Optional; DefaultParameterValue(null)>] ?autoUnionToJsonObject : bool) =
+            [<Optional; DefaultParameterValue(null)>] ?autoUnionToJsonObject : bool,
+            /// <summary>When set to <c>false</c> the codec will throw on <c>null</c> strings. Use <c>string option</c> to allow nulls.
+            [<Optional; DefaultParameterValue(null)>] ?allowNullStrings : bool) =
+
+        let defaultConverters: JsonConverter array =
+            if allowNullStrings = Some false then [| RejectNullStringConverter() |] else Array.empty
```

Comment:
I know I used this, and find it very legible
But FYI this syntax constructs a FSharpOption
Does not truly matter here but maybe
```suggestion
            if defaultArg allowNullStrings true then [| RejectNullStringConverter() |] else Array.empty
```
(please don't use)